pkg/controller/namespace/namespace_controller.go

```diff
@@ -145,6 +145,8 @@ func (r *ReconcileNamespace) Reconcile(request reconcile.Request) (reconcile.Res
 					reqLogger.Error(err, failedToCreateRoleBindingMsg)
 					return reconcile.Result{}, err
 				}
+				roleBindingName := permission.ClusterRoleName + "-" + subjectPermission.Spec.SubjectName
```

Comment:
`fmt.Sprintf("%s-%s", permission.ClusterRoleName, subjectPermission.Spec.SubjectName)` same for the next one please.